Source/swiftlint/Helpers/CompilerArgumentsExtractor.swift

```diff
@@ -1,14 +1,20 @@
 import Foundation
 import SourceKittenFramework
 
+private let escapedSpacePlaceholder = "\u{0}"
+
 struct CompilerArgumentsExtractor {
     static func allCompilerInvocations(compilerLogs: String) -> [String] {
         var compilerInvocations = [String]()
         compilerLogs.enumerateLines { line, _ in
             if let swiftcIndex = line.range(of: "swiftc ")?.upperBound, line.contains(" -module-name ") {
-                let invocation = line[swiftcIndex...]
+                let invocation = line
+                    .replacingOccurrences(of: "\\ ", with: escapedSpacePlaceholder)[swiftcIndex...]
                     .components(separatedBy: " ")
+                    .map { $0.replacingOccurrences(of: escapedSpacePlaceholder, with: " ") }
                     .expandingResponseFiles
+                    .map { $0.replacingOccurrences(of: "\\ ", with: escapedSpacePlaceholder) }
+                    .map { $0.replacingOccurrences(of: " ", with: escapedSpacePlaceholder) }
```

Comment:
To get this to work for us in the short term, this fix got a little out of control. 

Because the invocations get "expanded" by reading/appending the `.SwiftFileList` files, which also may contain spaces, another escaping sequence needs to happen. 

There is almost certainly a more elegant way of achieving this, which I'll hopefully flush out.